servers/src/main/java/tachyon/worker/block/evictor/LRUEvictor.java

```diff
@@ -32,21 +32,32 @@
 import tachyon.worker.block.BlockStoreEventListenerBase;
 import tachyon.worker.block.BlockStoreLocation;
 import tachyon.worker.block.meta.BlockMeta;
+import tachyon.worker.block.meta.StorageDir;
+import tachyon.worker.block.meta.StorageTier;
 
 public class LRUEvictor extends BlockStoreEventListenerBase implements Evictor {
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
   private final BlockMetadataManager mMeta;
 
   /**
-   * access-ordered {@link java.util.LinkedHashMap} from blockId to {@code true}, acts as a LRU
-   * double linked list where most recently accessed element is put at the tail while least recently
-   * accessed element is put at the head
+   * access-ordered {@link java.util.LinkedHashMap} from blockId to {@code true}(just to occupy the
+   * value), acts as a LRU double linked list where most recently accessed element is put at the
+   * tail while least recently accessed element is put at the head.
    */
   private Map<Long, Boolean> mLRUCache = Collections
       .synchronizedMap(new LinkedHashMap<Long, Boolean>(200, 0.75f, true));
```

Comment:
Could you either comment what `0.75f` and `true` represent in this constructor, or define them in static final member variables and pass these variables into this constructor.